elasticsearch/client/__init__.py

```diff
@@ -1311,8 +1311,20 @@ def scroll(self, body=None, params=None):
             are in multiple versions (7.0 and 6.latest)
         """
 
+        if scroll in SKIP_IN_PATH and body in SKIP_IN_PATH or \
+                scroll_id in SKIP_IN_PATH and body in SKIP_IN_PATH:
+            raise ValueError("You need to supply (scroll and scroll_id) or body.")
```

Comment:
It looks like the CI tests are failing due to this exception being of the wrong type. I can dig into this a bit further to understand what's going on.

Reply:
Hi @Winterflower . Thanks for being receptive to this PR. :)

The CI failure was my fault -- I didn't fully understand the `query_params` implementation, and my safety check on lines 1314 and below effectively re-implemented that existing logic.

The PR is much simpler now: just a one-line change in total. Thanks for your help!